src/main/java/io/temporal/samples/dsl/models/Workflow.java

```diff
@@ -0,0 +1,8 @@
+package io.temporal.samples.dsl.models;
+
+import java.util.Map;
+
+public class Workflow {
```

Comment:
Please rename to something like DslWorkflow as Workflow class is already one of the core DSL APIs.

Reply:
Updated